### PR TITLE
Home task (Task 4) | add edit/delete/add movie popUps

### PR DESCRIPTION
**Pull request by AnastasiiaDiachenko**

### CODE REVIEW

File: adiachenko/src/components/FilmForm/index.tsx [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+const initialFilm: initialFilmType = {
+    title: '',
+    release_date: '',
+    genres: [],
+    overview: '',
+    poster_path: '',
+    runtime: 0
+}
+
+const options = [
```
Comment (ygorbunkov) [inline adiachenko/src/components/FilmForm/index.tsx:44]: I didn't quite get the purpose of having 2 properties with exact same values. However, you may consider employing TypeScript enums:
```typescript
enum options {
   Fantasy = 'Fantasy'
   ..
}
```

File: adiachenko/src/components/FilmForm/index.tsx [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+    const resetForm = (event: SyntheticEvent) => {
+        event.preventDefault();
+        if (edit) {
+            setFilm(film);
+        } else {
+            setFilm(initialFilm);
+        }
+    };
+
+    const submitForm = (event: SyntheticEvent, save: boolean) => {
```
Comment (ygorbunkov) [inline adiachenko/src/components/FilmForm/index.tsx]: It looks like you can be more specific with something, like `FormEvent<HTMLFormElement>` in place of `SyntheticEvent`

File: adiachenko/src/components/FilmForm/index.tsx (hunk 1 above)
Comment (AnastasiiaDiachenko) [reply adiachenko/src/components/FilmForm/index.tsx:44]: To cover multiselect was desided to use 'react-select' npm package.
We need to use 2 same values becouse Select component from 'react-select' accept for each option value and label (and doesnt wark with only label/value or simple string)
Enum added

File: adiachenko/src/components/FilmForm/index.tsx (hunk 2 above)
Comment (AnastasiiaDiachenko) [reply adiachenko/src/components/FilmForm/index.tsx]: Was decided to use MouseEvent instead FormEvent<HTMLFormElement>
As this event are triggered onClick event on Button
`                {
                    edit ? (
                        <Button
                            onClick={(event) => submitForm(event, true)}
                        >Save</Button>
                    ) : (
                        <Button
                            onClick={(event) => submitForm(event, false)}
                        >Submit</Button>
                    )
                }`

**Pull request merged**